challenges/objects-arrays.js

```diff
@@ -54,12 +76,35 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
+function dobleEach(baseArray, basePropertyA, basePropertyB, newArray) {
+  for (let obj of baseArray) {
+    newArray.push(`${obj[basePropertyA]} ${obj[basePropertyB]}`);
+  }
+  return newArray;
+}
+
 const contactInfo = [];
+dobleEach(graduates, "first_name", "email", contactInfo);
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+function jankFilter(array, callback) {
+  let holdArray = []
+  for (let element of array) {
+    if (callback(element)) {
+      holdArray.push(element);
+    } else {
```

Comment:
this part is redundant, you can skip else { continue }, the result will be the same.

Reply:
We have all the universities now that contains "Uni", but we have to count them.
Let's also log how many we have.
👍